src/instruction.rs

```diff
@@ -34,6 +34,19 @@ pub enum EscrowInstruction {
 	Exchange {
 		/// the amount the taker expects to be paid in the other token
 		amount: u64
+	},
+
+	/// Allow initializer to cancel the trade 
+	/// Accounts expeted:
+	///
+	/// 0. `[signer]` The account of the person who initialized the escrow and wants to cancel
+	/// 1. `[writable]` The initializer's original token account that should get tokens back
+	/// 2. `[writable]` The escrow account, which should be closed after this tx
+	/// 3. `[]` The token program
+	/// 4. `[]` The PDA temp token account that has the tokens to return, should be closed
+	Cancel {
+		// the amount the initializer expects to get returned
+		amount: u64
```

Comment:
My motivation for this was in case there were multiple escrows happening between the same two accounts, but now that I think about it there will be different pdas for each escrow. I'll get rid of this